### ISSUE TITLE
Operration not supported

### ISSUE THREAD

**Issue by rickywu** [luckylz2git/exfat-synology#1]
I mount USB device at /volume1/usbexfat
and I can delete files successfully but can't copy and write data, just got error operation not supported, my DSM is 6.1.2

**Comment by luckylz2git** [comment]
Yes, this is a limitation of the system currently, and I still can not find a solution yet. If I find something in the future, I will update this repo.

Now I only use this to copy data from exfat usb to the NAS.

**Comment by rickywu** [comment]
@luckylz2git 
Can you check this repo https://github.com/dorimanx/exfat-nofuse

Linux non-fuse read/write kernel driver for the exFAT, FAT12, FAT16 and vfat (FAT32) file systems.
Maybe we can use this port to Synology DSM.

**Comment by luckylz2git** [comment]
Will take a loot later but no promise for anything.
If you try SSH or Telnet, data writing in exfat-partition works without any problems.

**Comment by mlukim** [comment]
I am not sure if it is same issue. After installing I see files and can download from file station, but cannot copy to other share (on same volume).
But I can copy using ssh - which is nice and for now enough :)

**Comment by joelstitch** [comment]
Has anyone found a fix for the issue? I also can access and delete files on the mount but cannot copy or move anything into the NAS unless I use SSH.

**Comment by rickywu** [comment]
MS will opensource ex-fat to Linux foundation, seems it shoud be a solution?
